files/scripts/startup_tsa_tsb.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+
+# Name: startup_tsa_tsb.py, version: 1.0
+#
+# Description: Module contains the definitions to the VOQ Startup TSA-TSB service
+
+import subprocess
+import sys, getopt
+from threading import Timer
+import os.path
+
+def getPlatform():
+    platform_key = "DEVICE_METADATA['localhost']['platform']"
+    platform = (subprocess.check_output(['sonic-cfggen', '-d', '-v', platform_key.replace('"',"'")]).strip()).decode()
+    return platform
+
+def getNumAsics():
+    platform = getPlatform()
+    asic_config_file = '/usr/share/sonic/device/{}/asic.conf'.format(platform)
+    file = open(asic_config_file, 'r')
+    Lines = file.readlines()
+    for line in Lines:
+       field = line.split('=')[0].strip()
+       if field == "NUM_ASIC":
+         return line.split('=')[1].strip()
+    return 0
+
+def getTsbTimerInterval():
+    platform = getPlatform()
+    conf_file = '/usr/share/sonic/device/{}/startup-tsa-tsb.conf'.format(platform)
+    file = open(conf_file, 'r')
+    Lines = file.readlines()
+    for line in Lines:
+       field = line.split('=')[0].strip()
+       if field == "STARTUP_TSB_TIMER":
+         return line.split('=')[1].strip()
+    return 0
+
+def getSonicConfig(ns, config_name):
```

Comment:
will this work for single asic linecards?

Reply:
Modified the functions to work for both single asic and multi asic